src/scss/components/list/_index.scss

```diff
@@ -1,34 +1,36 @@
-@import (reference) 'components/grid/index';
+@import "components/grid/index";
 
 #List {
-  .reset() {
+  @mixin reset() {
     margin: 1em 0;
   }
 }
```

Comment:
Please remove the entire `#List` block (lines 3-7).

Given that this file is the only one that used the mixin _and_ because you've already replaced its usage with its output on line 10, the block is useless.